src/Fable.Transforms/Python/Replacements.fs

```diff
@@ -2231,7 +2231,7 @@ let sets (com: ICompiler) (ctx: Context) r (t: Type) (i: CallInfo) (thisArg: Exp
 
         let args = injectArg com ctx r "Set" mangledName i.GenericArgs args
 
-        Helper.LibCall(com, "set", mangledName, t, args, i.SignatureArgTypes, ?thisArg = thisArg, ?loc = r)
+        Helper.LibCall(com, "set", mangledName, t, args, i.SignatureArgTypes, isInstance=not isStatic, ?thisArg = thisArg, ?loc = r)
```

Comment:
@alfonsogarciacaro I stand corrected, perhaps one of optional `LibCall` arguments `isInstance` is not necessary, if we're also passing `?thisArg`, we can derive it in the helper body, no?

Reply:
Ah, you're right 😅 Thanks!